### ISSUE TITLE
JavaScript implementation exits normally

### ISSUE THREAD

**Issue by ghost** [BlitzKraft/useless#6]
The JavaScript implementation terminates using `process.exit(0)`, which is simply a normal termination. It should send a `SIGTERM` to itself.

**Comment by BlitzKraft** [comment]
I agree. I am no js expert. I didn't know if it was even possible to send a SIGTERM via js. If you can, submit a pull request.
